experiment/measurer/coverage_utils.py

```diff
@@ -293,3 +295,42 @@ def extract_covered_regions_from_summary_json(summary_json_file):
     except Exception:  # pylint: disable=broad-except
         logger.error('Coverage summary json file defective or missing.')
     return covered_regions
+
+
+def extract_segments_and_functions_from_summary_json(  # pylint: disable=too-many-arguments
+        cov_summary_file, benchmark, fuzzer, trial_id, time, segment_df,
+        function_df, recorded_segments):
+    """Return a trial-specific data frame container with segment and function
+     coverage information given a trial-specific coverage summary json file."""
+
+    try:
+        coverage_info = get_coverage_infomation(cov_summary_file)
+        for function_data in coverage_info['data'][0]['functions']:
+            entry = pd.Series([
+                benchmark, fuzzer, trial_id, function_data['name'],
+                function_data['count'], time
+            ],
+                              index=function_df.columns)
```

Comment:
Should this go into the line above?

Reply:
I corrected it, but `make format` does this again.

Reply:
Adding `pylint: disable-all` helps me maintain the formatting.

Reply:
removed unnecessary `pylint` annotations.